drivers/webdav/onitu_webdav/tests/driver.py

```diff
@@ -0,0 +1,76 @@
+import os
+import hashlib
+
+from io import BytesIO
```

Comment:
I’m developing with py2 and the tests are OK. How can it be not compatible with py2? =